### PR TITLE
[PUI] Added AboutInventreeModal

### PR DESCRIPTION
**Pull request by matmair**
This PR adds a modal to view the system information, similar to the implementation in CUI.

Contains https://github.com/inventree/InvenTree/pull/5810

### CODE REVIEW

**Comment by SchrodingersGat** [comment]
LGTM but hard to check on netlify as the API endpoint is not available!

**Comment by matmair** [comment]
@SchrodingersGat  here are screenshots of the new section; to really check one would have to pull and run `inv server & inv frontend-dev` to check fullstack changes

**Comment by matmair** [comment]
@SchrodingersGat  here are screenshots of the new section; to really check one would have to pull and run `inv server & inv frontend-dev` to check fullstack changes

File: src/frontend/src/components/items/CopyButton.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  value: any;
+  label?: JSX.Element;
+}) {
+  return (
+    <MantineCopyButton value={value}>
+      {({ copied, copy }) => (
+        <Button
+          color={copied ? 'teal' : 'gray'}
+          onClick={copy}
+          title={t`copy to clipboard`}
```
Comment (SchrodingersGat) [inline src/frontend/src/components/items/CopyButton.tsx:18]: Please capitalise this `Copy to clipboard`

File: src/frontend/src/components/modals/AboutInvenTreeModal.tsx [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+            <Trans>Up to Date</Trans>
+          </Badge>
+        ) : (
+          <Badge color="teal">
+            <Trans>Update Available</Trans>
+          </Badge>
+        )}
+      </Group>
+      <Title order={5}>
+        <Trans>Server Versions</Trans>
```
Comment (SchrodingersGat) [inline src/frontend/src/components/modals/AboutInvenTreeModal.tsx:101]: I think `Version Information` serves better here

File: src/frontend/src/components/modals/AboutInvenTreeModal.tsx [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+            ],
+            data.links,
+            true
+          )}
+        </tbody>
+      </Table>
+      <Group>
+        <CopyButton
+          value={copyval}
+          label={<Trans>copy version information</Trans>}
```
Comment (SchrodingersGat) [inline src/frontend/src/components/modals/AboutInvenTreeModal.tsx:166]: Capitalize to `Copy version information`

**Review by SchrodingersGat — CHANGES_REQUESTED** [review]

**Comment by SchrodingersGat** [comment]
@matmair have checked out locally, looks very nice. I really like the visual flourishes on the button collar changes. Only a couple of very minor changes regarding text, other than that LGTM :)

File: src/frontend/src/components/items/CopyButton.tsx (hunk 1 above)
Comment (matmair) [reply src/frontend/src/components/items/CopyButton.tsx:18]: done

File: src/frontend/src/components/modals/AboutInvenTreeModal.tsx (hunk 2 above)
Comment (matmair) [reply src/frontend/src/components/modals/AboutInvenTreeModal.tsx:101]: I have changed it... not sure if this title is really informative though. `Version Information` seems very generic, maybe `Instance Versions` might be also a good fit?

File: src/frontend/src/components/modals/AboutInvenTreeModal.tsx (hunk 3 above)
Comment (matmair) [reply src/frontend/src/components/modals/AboutInvenTreeModal.tsx:166]: Done

**Review by SchrodingersGat — APPROVED** [review]

**Comment by SchrodingersGat** [comment]
@matmair thanks for the addition

**Pull request merged**